workflow/scripts/osemosys_global/powerplant/renewable_targets.py

```diff
@@ -0,0 +1,212 @@
+"""Function to set renewable targets."""
+import pandas as pd
+import itertools
+
+from data import get_years
+
+def apply_re_pct_targets(re_targets, remove_nodes, oar_df, renewables_list, 
+                     fuel_set, annual_demand_df, region_name):
+    
+    """Apply Renewable Energy targets by country, year and technology in 
+    relative terms compared to overall generation (= specified demand)."""
+    
+    if not remove_nodes:
+        remove_nodes = []
+        
+    node_dict = {}
+    target_techs_dict = {}
+    type_dict = {}
+    first_year_dict = {}
+    final_year_dict = {}
+    value_dict = {}    
+    accumulated_annual_demand = pd.DataFrame(columns = ["REGION", "FUEL", 
+                                                        "YEAR", "VALUE"])
+    
+    # Check if any targets are defined.
+    if not re_targets is None:
+        # Get transmission parameters from config file.
+        for target, target_params in re_targets.items():
+            node_dict[target] = target_params[0]
+            target_techs_dict[target] = target_params[1]
+            type_dict[target] = target_params[2]
+            first_year_dict[target] = target_params[3]
+            final_year_dict[target] = target_params[4]
+            value_dict[target] = target_params[5]
+        
+        # Per entry check if they are relative targets ('PCT').
+        for target in re_targets.keys():
+            if type_dict[target] == 'PCT':
```

Comment:
Not a required change, but an easy way to dedent this a level is to check for the opposite value! ie. 

```
if type_dict[target] != "PCT":
    continue 
# remaining code a indent level up
```

Reply:
Will just improve readability when we (eventually) and in standard formatting with line lengths! :)